spec/requests/portfolio_items_spec.rb

```diff
@@ -294,6 +294,9 @@
 
       it 'updates the field that is null' do
         expect(json["workflow_ref"]).to eq nullable_attributes[:workflow_ref]
+        expect(json["description"]).to eq nullable_attributes[:description]
```

Comment:
@syncrou Shouldn't this be that the key doesn't exist.

Reply:
No because the parser doesn't remove any nils, it just allows them through as nil if they are nil and the `OpenAPI::Nullable` validator is applied.

Reply:
@syncrou I think this is the response coming back from the API. The parser only validates the request but not the response. Have we checked if the response payload can send back nils or an empty string ""

Reply:
I added defaults that were not null, so now the response is correctly showing they were modified to null.